index.md

```diff
@@ -3,129 +3,130 @@ layout: home
 
-### Be friendly and patient
+### Будьте дружелюбны и терпеливы
 
-We understand that everyone has different levels of experience or knowledge in many diverse fields, be it technical or nontechnical in nature. We also have areas of knowledge we are eager to expand. We want to be a community where people can not only contribute, but feel comfortable to ask questions as well and learn along the way. When correcting another participant, respond with patience and try to keep it polite and civil. Remember that we all were newbies at one point.
+Мы понимаем, что у всех разный уровень опыта или знаний в различных областях, технических или нетехнических по своей природе. У нас также есть области знаний, которые мы хотим расширить. Мы хотим быть сообществом, в котором люди могут не только вносить свой вклад, но и чувствовать себя комфортно, задавая вопросы и попутно обучаясь. Исправляя другого участника, проявляйте терпение и старайтесь сохранять вежливость. Помните, что все мы когда-то были новичками.
 
-### Be inclusive
+### Будьте инклюзивны
 
-We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to, members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+Мы стремимся быть сообществом, которое приветствует и поддерживает людей любого происхождения и идентичности. Это включает, помимо прочего, представителей любой расы, этнической принадлежности, культуры, национального происхождения, цвета кожи, иммиграционного статуса, социального и экономического класса, уровня образования, пола, сексуальной ориентации, гендерной идентичности и выражения, возраста, размера, семейного положения, политических убеждений, религий, ментальных и физических способностей.
 
-### Be considerate
+### Будьте внимательными
 
-Your work will be used by other people, and you in turn will depend on the work of others. Any decision you make will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we're a world-wide community, so you might not be communicating in someone else's primary language.
+Ваша работа будет использоваться другими людьми, а вы, в свою очередь, будете зависеть от работы других. Любое принятое вами решение повлияет на пользователей и коллег, и вы должны учитывать эти последствия при принятии решений. Помните, что мы — международное сообщество, <?>.
```

Comment:
> so you might not be communicating in someone else's primary language.

need help with translation

Reply:
I think that this should be rephrased somehow

Reply:
heck it reads a bit odd in english too. Full disclosure, as you know, I speak russian like a 5 year old and read very slowly. How about:

`Your work will be used by other people, and you in turn will depend on the work of others.` reworded as 
` Your work will be used by other people, in the same way as you will depend on the work of others`:

`Ваша работа будет использоваться другими людьми, так же, как вы будете зависеть от работы других. 

"Users" in russian sounds really strange to me, but it is also because I have never spoken technically in russian.. I made it more specific as a result:

`Any decision you make will affect users and colleagues, and you should take those consequences into account when making decisions.` is updated to `Any decision you make will affect JavaScript Programmers, and your TC39 colleagues, and you should take those consequences into account.`

`Любое ваше решение повлияет на программистов JavaScript и ваших коллег по TC39, и вы должны принять во внимание эти последствия.`

For the last sentence I needed a heavy dose of machine translation help, so it might be totally off:

`Язык комитета - английский, но состоит из международных членов. Важно учитывать, что английский язык не для всех является основным языком, и что случаются недоразумения.`

Maybe?